src/scm/bitbucket.js

```diff
@@ -1,4 +1,277 @@
 "use strict";
 
 class Bitbucket {
-}
\ No newline at end of file
+  constructor(baseUrl, user, token) {
+    this.baseUrl = baseUrl;
+    this.user = user;
+    this.token = token;
+    this.accessToken = null;
+  }
+
+  get name() {
+    return 'bitbucket';
+  }
+
+  get canUseGist() {
+    return false;
+  }  
+
+  getAccessToken() {
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: 'https://bitbucket.org/site/oauth2/access_token',
+        headers: {
+          Authorization: `Basic RmZIVE02ZnN5NDJQQlJDRjRQOmVDZDN0TTh5TUpUeTJSMld4bTJWUzZoYWVKdnpuNzdw`
+        },
+        method: 'POST',
+        dataType: 'json',
+        contentType: 'application/x-www-form-urlencoded',
+        data: {
+          grant_type: 'refresh_token',
+          refresh_token: this.token
+        }
+      })
+      .then(resolve)
+      .fail(reject)
+    })
+    .then(response => {
+      chrome.storage.sync.set({ token: response.refresh_token });
+      this.accessToken = response.access_token;
+      return response.access_token;
+    })
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/77d86c0cbe12e28b49d988b7fdf8d52ef89bd093/src/scm/bitbucket.js#L42"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>